specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/2022-05-31/iotcentral.json

```diff
@@ -0,0 +1,3968 @@
+          }
+        }
+      }
+    },
+    "/organizations": {
+      "get": {
+        "operationId": "Organizations_List",
+        "summary": "Get the list of organizations the user has access to in an application",
+        "x-ms-examples": {
+          "List organizations": {
+            "$ref": "./examples/organizations_list.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          }
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/OrganizationCollection"
+            }
+          },
+          "default": {
+            "description": "An error response received from the IoT Central Service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/organizations/{organizationId}": {
+      "get": {
+        "operationId": "Organizations_Get",
+        "summary": "Get an organization by ID",
+        "x-ms-examples": {
+          "Get organization": {
+            "$ref": "./examples/organizations_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "in": "path",
+            "name": "organizationId",
+            "description": "Unique ID of the organization.",
+            "type": "string",
+            "maxLength": 48,
+            "pattern": "^(?!-)[a-z0-9-]{1,48}[a-z0-9]$",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Organization"
+            }
+          },
+          "default": {
+            "description": "An error response received from the IoT Central Service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "Organizations_Create",
+        "summary": "Create an organization in the application",
+        "x-ms-examples": {
+          "Add or update organization": {
+            "$ref": "./examples/organizations_create.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "in": "path",
+            "name": "organizationId",
```

Comment:
Similarly, this appear to be an organization name. And you might want to add a pattern, as it is the input parameter.

Reply:
Added, update organizationId and userId as global parameter for share the definition.